clientconn.go

```diff
@@ -719,15 +719,14 @@ func (cc *ClientConn) switchBalancer(name string) {
 	}
 	grpclog.Infof("ClientConn switching balancer to %q", name)
 
-	if cc.dopts.balancerBuilder != nil {
-		grpclog.Infoln("ignoring balancer switching: WithBalancer DialOption used instead")
+	if strings.ToLower(cc.curBalancerName) == strings.ToLower(name) {
```

Comment:
I would rather move this above the info log on 720 also, but if you feel strongly then OK.

Reply:
Done